.gitignore

```diff
@@ -1,6 +1,9 @@
 *.swp
 *.beam
```

Comment:
(I know this isn't realted to this PR but this cuased me to notice) - BEAM? As in Erlang BEAM? Is that actually useful in a Rust repo?